Resources/html/views/commit/commit.js

```diff
@@ -5,16 +5,18 @@ var contextLines = 0;
 
 var showNewFile = function(file)
 {
-	setTitle("New file: " + file.path.escapeHTML());
+	setTitle("New file: " + file.path);
+    diff.innerHTML = "";
```

Comment:
Nitpick: indentation. This file seems to be mostly tab-based, so I'd prefer if it stayed that way 😉.

Reply:
Whoops, I thought I caught all of these. Thanks.